### ISSUE TITLE
Classes contain structs

### ISSUE THREAD

**Issue by DialLuesebrink** [TNG/ArchUnitNET#96]
I would expect this test not to fail in a code base with classes and structs.
It would be nice to have Structs separately.

```csharp
using ArchUnitNET.Fluent;
using ArchUnitNET.NUnit;
using NUnit.Framework;
using static ArchUnitNET.Fluent.ArchRuleDefinition;

namespace PostSolutionTests
{
    class ClassesShouldNotBeStructs
    {
        private static readonly IArchRule ClassesShouldNotBeStructsRule = Classes().Should().NotBeStructs();

        [Test, Category("ArchUnitTest")]
        public static void ClassesShouldNotBeStructsTests() => ClassesShouldNotBeStructsRule.Check(Architecture.Dialux);
    }
}
```

**Comment by fgather** [comment]
Thanks for the valuable input. I'd assume that we have the same problem with records.

**Comment by DialLuesebrink** [comment]
Afaik records are classes internally, though there might be some way to discern them through some attributes or so.

**Comment by fgather** [comment]
Released in 0.9.0
